tests/test_pipeline.py

```diff
@@ -1,5 +1,70 @@
 from pyannote.pipeline import Pipeline
+from pyannote.pipeline.parameter import Uniform, Integer, ParamDict, ParamList
+from tests.utils import FakeTrial
 
 
-def test_params():
-    assert 2 == 1 + 1
+def test_pipeline_params_simple():
```

Comment:
Would be nice to also add tests that actually run the optimizer...

Reply:
I made some very rudimentary tests, if you have any suggestions i'd be glad to know